rlclientlib/live_model_impl.cc

```diff
@@ -243,6 +243,61 @@ namespace reinforcement_learning {
     return error_code::success;
   }
 
+  int live_model_impl::request_multi_slot_decision(const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, api_status* status)
+  {
+	  const auto uuid = boost::uuids::to_string(boost::uuids::random_generator()());
+	  return request_multi_slot_decision(uuid.c_str(), context_json, flags, resp, status);
+  }
+
+  int live_model_impl::request_multi_slot_decision(const char * event_id, const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, api_status* status)
+  {
+	  if (_learning_mode == APPRENTICE || _learning_mode == LOGGINGONLY) {
```

Comment:
Looks like lines 254-290 are copy-paste from 201-236. Can we move it to some private method?